src/ignitus-Api/constants.ts

```diff
@@ -4,7 +4,7 @@ export const TESTIMONIAL_URL = `${BASE_URL}/testimonials`;
 export const STUDENT_SIGN_UP = `${BASE_URL}/register/student`;
 export const STUDENT_SIGN_IN = `${BASE_URL}/login`;
 
-export const FRONTEND_CONTRIBUTOR_API = 'https://api.github.com/repos/Ignitus/Ignitus-Client-Side-Development/contributors?per_page=14';
+export const FRONTEND_CONTRIBUTOR_API = 'https://api.githubb.com/repos/Ignitus/Ignitus-Client-Side-Development/contributors?per_page=14';
```

Comment:
Please revert it in the last commit once we approve the PR, its good for testing :) Thanks.